src/components/Quiz/Quiz.stories.tsx

```
@@ -0,0 +1,355 @@
+              areaKnowledgeId: 'matematica',
+              subjectId: 'algebra',
+              topicId: 'equacoes',
+              subtopicId: 'segundo-grau',
+              contentId: 'algebra'
+            }
+          ],
+          options: [
+            { id: 'opt1', option: 'x = 1 e x = 4' },
+            { id: 'opt2', option: 'x = 2 e x = 3' },
+            { id: 'opt3', option: 'x = 2 e x = 3' },
+            { id: 'opt4', option: 'x = -2 e x = -3' }
```

ESLint

[error] Insert `,`

(prettier/prettier)